src/visitor.js

```diff
@@ -25,11 +25,12 @@ function genVar(filename) {
 // VisitState holds the state of the visitor, provides helper functions
 // and is the `this` for the individual coverage visitors.
 class VisitState {
-    constructor(types, sourceFilePath) {
+    constructor(types, sourceFilePath, inputSourceMap) {
         this.varName = genVar(sourceFilePath);
         this.attrs = {};
         this.nextIgnore = null;
         this.cov = new SourceCoverage(sourceFilePath);
+        this.cov.inputSourceMap(inputSourceMap);
```

Comment:
Perhaps let's only assign `this.data.inputSourceMap` if `inputSourceMap` is `undefined`? that way, for anyone not using the new API feature, the shape of the coverage object stays identical 👍